main.js

```diff
@@ -0,0 +1,12 @@
+document.querySelector("span").remove()
+document.querySelector("li").remove()
+
+const removeH3 = document.querySelectorAll("main .posts article");
+for (let rem of removeH3) {
+    if (rem.textContent.includes("Stop Planning")){
+        rem.remove();
+    }
+};
```

Comment:
Interesting way to do this one! I like that it will always target the "Stop Planning" article no matter its position in the DOM. 
Here's another way to do this if removing the **_last element_** specifically is the main priority:
`h3[h3.length - 1].remove()`